<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>命令客户端</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>51</width>
            <height>16</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>宋体</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>IP地址</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="minimumSize">
           <size>
            <width>251</width>
            <height>31</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>131</width>
            <height>16</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>宋体</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>请输入执行命令</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="horizontalGroupBox">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pushButton_connect">
           <property name="minimumSize">
            <size>
             <width>91</width>
             <height>41</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>107</width>
             <height>41</height>
            </size>
           </property>
           <property name="text">
            <string>发送</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_clear">
           <property name="minimumSize">
            <size>
             <width>91</width>
             <height>41</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>106</width>
             <height>41</height>
            </size>
           </property>
           <property name="text">
            <string>清空</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_collect">
           <property name="minimumSize">
            <size>
             <width>91</width>
             <height>41</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>106</width>
             <height>41</height>
            </size>
           </property>
           <property name="text">
            <string>收藏</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>宋体</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>命令列表</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_autosend">
          <property name="text">
           <string>自动发送</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget_cmdlist"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
